src/configs/index.js

```diff
@@ -13,3 +16,84 @@ require('./bull')()
 
 const path = require('path')
 global.PROJECT_ROOT_DIRECTORY = path.join(__dirname, '..')
+;(function () {
+	const schema = {
```

Comment:
@nevil-mathew - As discussed this can be moved out from here. Remove the validation logic. We can have a default schema

Reply:
Removed